deps/streamsearch/sbmh.js

```diff
@@ -134,33 +141,31 @@ SBMH.prototype._sbmh_feed = function(data) {
 
     if (pos >= 0) {
       // Discard lookbehind buffer.
-      this.emit('info', false, lookbehind, 0, this._lookbehind_size);
+      this.emit('info', false, this._lookbehind, 0, this._lookbehind_size);
       this._lookbehind_size = 0;
     } else {
       // Cut off part of the lookbehind buffer that has
       // been processed and append the entire haystack
       // into it.
-      var bytesToCutOff = this._lookbehind_size + pos;
-
+      const bytesToCutOff = this._lookbehind_size + pos;
       if (bytesToCutOff > 0) {
         // The cut off data is guaranteed not to contain the needle.
-        this.emit('info', false, lookbehind, 0, bytesToCutOff);
+        this.emit('info', false, this._lookbehind, 0, bytesToCutOff);
```

Comment:
this is the last line, which is not covered by a unit test... And i cant figure out, when this line is hit.